dev/config/monitored-pod/configmap.yaml

```diff
@@ -14,7 +14,7 @@ data:
     </style>
     </head>
     <body>
-    <h1>Welcome to nginx!</h1>
+    <h1>Welcome-updated-updated-updated-updated to nginx!</h1>
```

Comment:
Same with above. This file is updated during the k8s-test, but it shouldn't be committed.

Reply:
Yes.